config/env.example

```diff
@@ -45,7 +45,8 @@ SENTRY_DSN_JS=""
 MATOMO_ENABLED="disabled"
 MATOMO_ID="73"
 
-SENDINBLUE_ENABLED="disabled"
+SENDINBLUE_ENABLED="enabled"
+CRISP_ENABLED="enabled"
```

Comment:
Dans le `env.example`, on désactive par défaut les services optionels (à plus forte raison s'ils ont besoin d'identifiants qu'on ne veut pas mettre dans le fichier).

```suggestion
CRISP_ENABLED="disabled"
```

Reply:
C'est noté